lightning/src/onion_message/messenger.rs

```diff
@@ -265,14 +265,31 @@ impl Responder {
 		}
 	}
 
-	/// Creates the appropriate [`ResponseInstruction`] for a given response.
+	/// Creates a [`ResponseInstruction::WithoutReplyPath`] variant of [`ResponseInstruction`]
+	/// for a given response.
+	///
+	/// [`ResponseInstruction::WithoutReplyPath`] indicates that the response should be sent
+	/// without creating a reply path for the receiver to use to send back a response to us.
```

Comment:
This part isn't necessary as it simply repeats the linked docs. Generally, a short summary may be fine, but we should avoid duplicating the docs otherwise they risk getting out of sync.

Reply:
That makes sense!
Addressed in [8d5f9dc](https://github.com/lightningdevkit/rust-lightning/commit/8d5f9dce49b9b2b49a2e67d52a3021325ac3fcb3)
Thanks a lot for the suggestion